modules/network-flow-matrix.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * installing/install_config/configuring-firewall.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="network-flow-matrix_{context}"]
+= {product-title} network flow matrix
+
+The network flow matrix describes the ingress flows to {product-title} services.
+The network information in the matrix is accurate for both bare-metal and cloud environments.
+Use the information in the network flow matrix to help you manage ingress traffic.
+You can restrict ingress traffic to essential flows only to improve network security.
+
+To view or download the raw CSV content, see link:https://raw.githubusercontent.com/rohennes/openshift-docs/TELCODOCS-1670/snippets/network-flow-matrix.csv[this resource].
+
+Additionally, consider the following dynamic port ranges when managing ingress traffic:
+
+* `9000-9999`: General use ports for system services.
```

Comment:
Host level services
(instead of 'General use ports for system services')

the term "general use" might be confusing. I dont think it's wise to use these outside the platform itself

Reply:
Thanks Yuval, I updated he PR accordingly